DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.Login
 @model ChooseACentreViewModel
 @{
-  ViewData["Title"] = "Choose A Centre";
+  ViewData["Title"] = "Choose a centre";
```

Comment:
This doesn't really matter, but I don't think there's an issue with the tab title's capitalisation - especially because (confusingly) every work is usually capitalised there :P